docs/sources/microsoft-365/msft-teams/example-api-responses/sanitized/Users_onlineMeetings_v1.0.json

```diff
@@ -0,0 +1,57 @@
+{
+  "@odata.type":"#microsoft.graph.onlineMeeting",
+  "autoAdmittedUsers":"everyone",
+  "audioConferencing":{
+    "tollNumber":"5552478",
+    "tollFreeNumber":"5550588",
+    "ConferenceId":"9999999",
+    "dialinUrl":"https://dialin.teams.microsoft.com/6787A136-B9B8-4D39-846C-C0F1FF937F10?id=xxxxxxx"
+  },
+  "chatInfo":{
+    "@odata.type":"#microsoft.graph.chatInfo",
+    "threadId":"19:cbee7c1c860e465cebf7bee0d@thread.skype",
+    "messageId":"153367081"
+  },
+  "creationDateTime":"2018-05-30T00:12:19.0726086Z",
+  "endDateTime":"2018-05-30T01:00:00Z",
+  "id":"112f7296-5fa4-42ca-bae8-6a692b15d4b8_19:cbee7c1c860e465f8258e3cebf7bee0d@thread.skype",
+  "joinWebUrl":"https://teams.microsoft.com/l/meetup-join/19%3a:meeting_NTg0NmQ3NTctZDVkZC00YzRhLThmNmEtOGQDdmZDZk@thread.v2/0?context=%7b%22Tid%22%3a%aa67bd4c-8475-432d-bd41-39f255720e0a%22%2c%22Oid%22%3a%22112f7296-5fa4-42ca-bb15d4b8%22%7d",
+  "participants":{
+    "@odata.type":"#microsoft.graph.meetingParticipants",
+    "attendees":[
+      {
+        "@odata.type":"#microsoft.graph.identitySet",
+        "identity":{
+          "user":{
+            "@odata.type":"#microsoft.graph.identity",
+            "id":"112f7296-5ca-bae8-6a692b15d4b8"
+          }
+        },
+        "upn":"upn-value"
+      }
+    ],
+    "organizer":{
+      "@odata.type":"#microsoft.graph.identitySet",
+      "identity":{
+        "user":{
+          "@odata.type":"#microsoft.graph.identity",
+          "id":"5810cedeb-b2c1-e9bd5d53ec96"
+        }
+      },
+      "upn":"upn-value"
+    }
+  },
+  "startDateTime":"2018-05-30T00:30:00Z",
+  "videoTeleconferenceId":"123456789",
+  "lobbyBypassSettings":{
+    "scope":"everyone",
+    "isDialInBypassEnabled":true
+  },
+  "joinMeetingIdSettings":{
+    "isPasscodeRequired":false,
+    "joinMeetingId":"1234567890",
+    "passcode":null
```

Comment:
redact, if it's not (unclear from this example - I'd prefer node being deleted from JSON entirely

Reply:
I'd leave the object only with `joinMeetingId`, the other fields are not important and can be redacted.

Reply:
Done.